ansible/roles/postgres-init/tasks/main.yml

```diff
@@ -31,3 +31,15 @@
 
     - name: Postgres                   | Create Postgres data cluster
       command: su postgres -c "/usr/pgsql-11/bin/initdb -D /srv/postgres"
+
+    - name: PMM                        | Fix postgresql config
```

Comment:
- name: Postgres

And please add a short and clear description like  "| Change path to  postgresql.log'. It will help us in the future during the debug

Reply:
Done